app/src/main/res/drawable/state_button_primary_background.xml

```diff
@@ -2,5 +2,8 @@
 <shape xmlns:android="http://schemas.android.com/apk/res/android"
   android:shape="rectangle">
   <corners android:radius="@dimen/state_previous_next_button_radius" />
-  <solid android:color="@color/oppia_primary" />
+  <solid android:color="@color/component_color_resume_lesson_activity_continue_button_background_color" />
+  <stroke
+    android:width="1dp"
+    android:color="@color/component_color_resume_lesson_activity_continue_button_border_color" />
```

Comment:
Rename this color to `component_color_continue_button_border_color`

This is done so that this particular style can be used generically anywhere in code.

Reply:
done.